test/scripts/personalization.jest.js

```diff
@@ -122,7 +122,7 @@ describe('Test personalization.js', () => {
   it('Show partner-level-platinum but don\'t show partner-level-gold block', () => {
     jest.isolateModules(() => {
       const cookieObject = {
-        SPP: {
+        DX: {
```

Comment:
@DejanBozic2DM @dejanpreradovic is it correct to read the cookie data from "DX"? I expected to still have separate items in the cookie for TPP and SPP